packages/api/types.ts

```diff
@@ -2288,3 +2331,12 @@ export type TSocketPaginateableEndpointNames = KeysMatching<
     TSocketEndpoints,
     { request: { limit?: number; offset?: number } }
 >;
+
+export type TLoginHistoryData = {
```

Comment:
can we move this type definition out from this api/types file.

Reply:
this is used only once. i have moved it local,to to the file login-history-content.tsx in which it is used. 👍🏼